README.md

```diff
@@ -134,3 +251,68 @@ It returns a text file in working directory.
     For different type of software versions the raw data path could be different for the same
     concept. For example, Nanonis software `generic 5e` has `/Piezo Configuration/Active Calib.`
     and generic 4.5 has `/Piezo Calibration/Active Calib.` for the same concept `/ENTRY[entry]/INSTRUMENT[instrument]/piezo_config/active_calib`.
+  - In the config file the concept that takes data from ELN is denoted by `@eln` otherwise data will come from experimental raw files.
+  - Importantly one should note the `NXdata` concept `/ENTRY[entry]/DATA[data]` it takes dict of the list. Each key of the dict refers to the `bais` and `current` in multiple given setup, saying, with and without `filter` check points. For other setup, one can extend the dict following the same convenstion used here.
+
+### Raw File
+This type of file (such as `example.dat`, `example.sxm`) just come from the experiment. 
+## Run the Reader from CLI
```

Comment:
Move this up so people know how to use this tool.

Reply:
Done!